documentation/docs/opal-plus/deploy.mdx

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 2
+title: Deploy OPAL+
+---
+
+With OPAL+, you get access to private Docker images that include additional features and capabilities.
```

Comment:
and higher level of support and SLA

Reply:
I've added this to the OPAL+ features, the line above is in the "deploy" section so it states only about technical features.